docs/tutorials/intro.md

```diff
@@ -130,13 +130,22 @@ In this tutorial we are going to use a synchronous file-system data provider whi
 
 We're going to use [JSON CLDR](https://github.com/unicode-cldr/cldr-json) as our source data. JSON CLDR is an export of [CLDR data](http://cldr.unicode.org/index/downloads) into JSON maintained by Unicode.
 
+We are also going to use Unicode property data shipped as a zip file in the ICU4C release.
+
 The `provider_fs` component has a binary application which will fetch the CLDR data and generate ICU4X data out of it.
 
 ```
 cd ~/projects/icu
+wget https://github.com/unicode-org/icu/releases/download/release-70-1/icuexportdata_uprops_full.zip
```

Comment:
Would love it if the tool could do this download directly

Reply:
Agreed. Tracked by #718, #1105